src/data/mv_polynomial.lean

```diff
@@ -2,12 +2,72 @@
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Johan Commelin, Mario Carneiro
-
-Multivariate Polynomial
 -/
 import algebra.ring
 import data.finsupp data.polynomial data.equiv.algebra
 
+/-!
+# Multivariate polynomials
+
+This file defines polynomial rings over a base ring (or even semiring),
+with variables corresponding to the terms in a general type σ (which could
```

Comment:
```suggestion
with variables from a general type σ (which could
```
IMHO the term "terms" suggests that we require some structure on `σ`, but we treat them as atoms.

Reply:
```suggestion
with variables from a general type `σ` (which could
```